### PR TITLE
Reload changed source files (integration test and KafkaNodeContainer)

### PR DESCRIPTION
**Pull request by ivan-abc**
“Signed-off-by: [Ivan Sysoev] [ivanabc97@gmail.com]”

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I have the right to submit it under the open source license indicated in the file; or

(b) The contribution is based upon previous work that, to the best of my knowledge, is covered under an appropriate open source license and I have the right under that license to submit that work with modifications, whether created in whole or in part by me, under the same open source license (unless I am permitted to submit under a different license), as indicated in the file; or

(c) The contribution was provided directly to me by some other person who certified (a), (b) or (c) and I have not modified it.

(d) I understand and agree that this project and the contribution are public and that a record of the contribution (including all personal information I submit with it, including my sign-off) is maintained indefinitely and may be redistributed consistent with this project or the open source license(s) involved.

### CODE REVIEW

**Comment by paulgrape** [comment]
Tests failed because of outdated gitlab-ci.yaml file.
It would be better to rebase last changes from master branch.

File: README.md [hunk 1]
```diff
@@ -64,3 +64,18 @@ Not supported
...
+| storage-driver-request-size | integer | 1048576 | The maximum size of a request in bytes. This setting will limit the number of record batches the producer will send in a single request to avoid sending huge requests. |
+| storage-driver-batch-size | integer | 16384 | Allows you to change the batch size |
+| storage-net-linger | integer | 0 | The delay before sending the records. This setting gives the upper bound on the delay for batching: once we get *batch.size* worth of records for a partition it will be sent immediately regardless of this setting, however if we have fewer than this many bytes accumulated for this partition we will 'linger' for the specified time waiting for more records to show up. |
+| storage-driver-buffer-memory | long | 33554432 | The total bytes of memory the producer can use to buffer records waiting to be sent to the server. |
+| storage-driver-compression-type | string | none | The compression type for all data generated by the producer. The default is none (i.e. no compression). Valid values are none, gzip, snappy, lz4, or zstd.  |
```
Comment (akurilov) [inline README.md]: let it be the default value to be `null` (meaning "none")
also, the type is enum rather than string

**Review by akurilov — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -64,3 +64,18 @@ Not supported
...
+| storage-driver-request-size | integer | 1048576 | The maximum size of a request in bytes. This setting will limit the number of record batches the producer will send in a single request to avoid sending huge requests. |
+| storage-driver-batch-size | integer | 16384 | Allows you to change the batch size |
+| storage-net-linger | integer | 0 | The delay before sending the records. This setting gives the upper bound on the delay for batching: once we get *batch.size* worth of records for a partition it will be sent immediately regardless of this setting, however if we have fewer than this many bytes accumulated for this partition we will 'linger' for the specified time waiting for more records to show up. |
+| storage-driver-buffer-memory | long | 33554432 | The total bytes of memory the producer can use to buffer records waiting to be sent to the server. |
+| storage-driver-compression-type | enum | null | The compression type for all data generated by the producer. The default is none (i.e. no compression). Valid values are none, gzip, snappy, lz4, or zstd.  |
```
Comment (akurilov) [inline README.md]: "Valid values are `null`, "gzip"..."

File: src/main/resources/defaults-storage-driver-kafka.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      key:
+        enabled: false
+    request:
+      size: 1048576
+    batch:
+      size: 16384
+    buffer:
+      memory: 33554432
+    compression:
+      type: none
```
Comment (akurilov) [inline src/main/resources/defaults-storage-driver-kafka.yml]: null

**Review by akurilov — CHANGES_REQUESTED** [review]

**Review by akurilov — APPROVED** [review]

**Pull request merged**